packages/backend/src/api/resources/SamlConnection.ts

```diff
@@ -0,0 +1,49 @@
+import type { AttributeMapping } from './AttributeMapping';
+import type { SamlConnectionJSON } from './JSON';
+
+export class SamlConnection {
+  constructor(
+    readonly id: string,
+    readonly name: string,
+    readonly domain: string,
+    readonly idp_entity_id: string,
+    readonly idp_sso_url: string,
+    readonly idp_certificate: string,
+    readonly idp_metadata_url: string,
+    readonly acs_url: string,
+    readonly sp_entity_id: string,
+    readonly sp_metadata_url: string,
+    readonly active: boolean,
+    readonly provider: string,
+    readonly user_count: number,
+    readonly sync_user_attributes: boolean,
+    readonly allow_subdomains: boolean,
+    readonly allow_idp_initiated: boolean,
+    readonly created_at: number,
+    readonly updated_at: number,
+    readonly attribute_mapping: AttributeMapping,
```

Comment:
These class properties need to be camelCased (not snaked_cased). 
In general, the convention we use is to use snake_case for the DTOs (a Data Transfer Object is an object that is used to encapsulate data, and send it from one subsystem of an application to another, eg from our backend to the customer's server using `@clerk/backend) and camelCase within the Clerk package.

To be more precise with an example, even though we use `idp_sso_url` to transfer the sso URL in the network layer, our customers need to be able to access it using:

```ts
const samlConnection = await apiClient.samlConnections.getSamlConnection(...);
samlConnection.idpSsoUrl;
```